HDSAviz/Basic_Plot_Updated.py

```diff
@@ -0,0 +1,130 @@
+"""This is a basic script for plotting an analysis file.
+
+    This script is just the first step in visualizing the data set. This file
+    comprises of the method creating an html bokeh plot and a command at the
+    end to implement the method.
+
+"""
+from collections import OrderedDict
+
+import numpy as np
+import pandas as pd
+import pdb
+
+from bokeh.plotting import figure, show, output_file
+
+# import data_processing as dp
+
+
+def make_plot(Dataframe, Cols, minvalues):
+    """Basic method to plot sensitivity anlaysis.
+
+    This is the method to generate a bokeh plot followung the burtin example
+    template at the bokeh website. For clarification, parameters refer to an
+    output being measured (Tmax, C, k2, etc.) and stats refer to the 1st or
+    total order sensitivity index.
+
+    Parameters:
+    --------------
+    Dataframe : Dataframe containing sensitivity analysis results to be plotted
+                {Name of csv file to be read into as a panda dataframe (String)
+                **Note file must be in same folder as script.**}
+                Delete upon change
+    Cols: Names of the columns to be plotted (Array of strings)
+    minvalues: Cutoff minimum for which parameters should be plotted (Array of
+                floats)
+    Returns:
+    p: Figure of the data to be plotted
+    """
+    # Read in csv file as panda dataframe.
+    tdf = pd.read_csv(Dataframe, delimiter=' ', skipinitialspace=True,
+                      engine='python')
+    df = tdf
+    # df = Dataframe
+    maxval = 0
+    # Remove rows which have values less than cutoff values
+    for i in range(0, minvalues.size):
+        df = df[df[Cols[i]] > minvalues[i]]
+        if maxval < max(df[Cols[i]]):
+            maxval = max(df[Cols[i]])
+    df = df.dropna()
+
+    # Create dictionary of bar colors, assume up to 3 stats are plotted
+    # per parameter.
+    colors = ["#0d3362", "#c64737", "black"]
+    stat_color = OrderedDict()
+    for i in range(0, Cols.size):
+        stat_color[i] = colors[i]
+    # Reset index of dataframe.
+    df = df.reset_index(drop=True)
+
+    # Sizing parameters
```

Comment:
# It is going to be Difficult but for ideas : 
### can sizing parameters be made automatic so as to adjust based on window size?? 
There is a [resizing tool in bokeh](http://bokeh.pydata.org/en/0.10.0/docs/user_guide/tools.html#resizetool) and javascript shall probably make it flexible to interact with resizing..... 

Default values are definitely helpful but might be  trouble